e2e/features/stepDefinitions/service/start-appeal.steps.ts

```diff
@@ -4,12 +4,24 @@ import { Given, Then } from 'cucumber';
 import { expect } from 'chai';
 
 const isfeePaymentEnabled = require('../../../ia.conf').isfeePaymentEnabled === 'true';
+const isOutOfCountryEnabled = require('../../../ia.conf').isOutOfCountryEnabled === 'true';
 const ccdFormPage = new CcdFormPage();
 const startAppealFlow = new StartAppealFlow();
 
 Given('I complete the `Tell us about your client` page', async function () {
     expect(await ccdFormPage.headingContains('Tell us about your client')).to.equal(true);
-    await startAppealFlow.completeScreeningQuestions(true);
+    if (isOutOfCountryEnabled) {
+        await startAppealFlow.completeScreeningQuestionsOutOfCountry(true);
+        expect(await ccdFormPage.headingContains('Tell us about your client')).to.equal(true);
```

Comment:
expect(await ccdFormPage.headingContains('Tell us about your client')).to.equal(true);

The above line is repeated.
The logic here need not change. This logic can be moved to start-appeal.flow.ts completeScreening Questions

Reply:
The line is repeated in 2 pages

Reply:
Code moved to startAppeal.flow.ts